### PR TITLE
fix: Disable deserializing shared ArrayBuffer views into MagicBuffer

### PR DESCRIPTION
**Pull request by andreubotella**
Part of #3.

### CODE REVIEW

**Comment by bartlomieju** [comment]
Looks ok to me as a work-around, I'll wait for @piscisaureus review before landing.

**Review by piscisaureus — APPROVED** [review]

**Pull request merged**
